packages/replayio/src/utils/recordings/types.ts

```diff
@@ -37,6 +37,27 @@ export type LogEntry = {
   recordingId?: string;
   server?: string;
   timestamp: number;
+  baseURL?: string;
+  targetContentHash?: string;
+  targetURLHash?: string;
+  targetMapURLHash?: string;
+  parentId?: string;
+  parentOffset?: number;
+};
```

Comment:
Man our log entries are kind of a gross grab-bag of junk 😆

Reply:
yeee, i was thinking of making them a discriminated union - that could help us to maintain this (and just give us more context on which property belongs to which log entry)

Reply:
How much of that would just be false confidence (given that we don't control Chromium/Node or the data they log) 😆

Reply:
I don't think it would have to be worse than what we have today. I didn't mean to remove the runtime validation that we already have (note: it's not exhaustive anyway).

Reply:
I don't think it would be _worse_ either. I'm just not sure it would be _better_ since we only control one side of this log.